README.md

```diff
@@ -96,6 +96,9 @@ parameters = {
 # Add a member to a list
 @mailgun.list_members.add "devs@your.mailgun.domain", "Q@mi6.co.uk"
 
+# Add multiple mailing list members to a list (limit 1,000 per call)
+@mailgun.list_members.add_multi "devs@your.mailgun.domain", [{"address": "Alice <alice@example.com>", "vars": {"age": 26}}, {"name": "Bob", "address": "bob@example.com", "vars": {"age": 34}}].to_json, {:upsert => true}
```

Comment:
@adamof  I just test it in my local ENV, and it do working well. So I think move inside is OK.